src/profile/components/__snapshots__/ProfilePage.test.jsx.snap

```diff
@@ -1135,7 +1135,7 @@ exports[`<ProfilePage /> Renders correctly in various states viewing own profile
                     className="certificate-type-illustration"
                     style={
                       Object {
-                        "backgroundImage": "url(null)",
+                        "backgroundImage": "url([object Object])",
```

Comment:
Is it weird that it's dumping an object into the URL for the snapshot?

Reply:
I think this is the normal behavior for the result of a loaded file? I'd bet this is jest's way of handling unknown webpack loaders.

Reply:
That might make sense - k.